common-app-lib/src/cmr/common_app/api/request_logger.clj

```diff
@@ -82,6 +82,18 @@
                                (assoc-in [:headers "Content-SHA1"] body-sha1))]
       updated-response)))
 
+(defn assoc-hashes
```

Comment:
can we rename this func to be more clear of what it does? something like 'add-hash-to-data'?

Reply:
glad you said something, I also don't like the name.